### DISCUSSION TITLE
Disable Plex Login

### DISCUSSION THREAD

**Discussion by EaterComputer** [sct/overseerr#1533] [Feature Requests]
I would appreciate an option to disable Plex Login. My family all use the same plex account credentials with different users to be able to get the skip intro button for all users. It would be great if you could force local user login.

**Comment by sct** [comment]
Maybe in the future when we decouple Plex and add other media servers, but no plans any time soon to do this.

May I recommend having them all create plex accounts and then adding those accounts as home users? They will get your plex pass feature this way but they will each have their own accounts.

**Reply by EaterComputer** [reply]
I was unaware of that. Thank you for mentioning it!

**Comment by alanman87** [comment]
I only use local accounts for Overseerr but my users forget this often and experience issues with sign in. It’s been over 2 years since this discussion was started. I see no updates on the rewrite. Until then it would be extremely helpful to just have a toggle to disable plex login. Please give us a work around for this to use in the mean time.
